workflows/microbiome/mags-building/MAGs-generation-tests.yml

```
@@ -0,0 +1,69 @@
+- doc: Test for Metagenome-Assembled-Genomes-(MAGs)-generation
+  job:
+    Trimmed grouped paired reads:
```

```suggestion
    Trimmed grouped:
```
that's redundant with what's shown in the workflow form.

---

I don't understand. This input are reads that are grouped. This allows for co-assembly. The workflow for grouping will also be submitted soon. If paired it not needed I would go for:
```suggestion
    Grouped trimmed reads:
```